packages/ui/src/views/repo/webhooks/webhook-create/components/create-webhooks-form-fields.tsx

```diff
@@ -15,7 +15,7 @@ import {
 import { TriggerEventsEnum, WebhookEvent, WebhookFormFieldProps, WebhookTriggerEnum } from '@/views'
 
 export const WebhookToggleField: FC<WebhookFormFieldProps> = ({ register, watch, setValue, t }) => (
-  <StackedList.Root borderBackground withoutBorder>
+  <StackedList.Root>
     <StackedList.Item
       className="!rounded px-5 py-3.5"
```

Comment:
```suggestion
      className="!rounded px-5 py-3"
```
We can reduce this indentation to get closer to the design height of this element